wagtail/admin/templates/wagtailadmin/pages/listing/_list.html

```diff
@@ -2,9 +2,7 @@
 {% load l10n %}
 {% load wagtailadmin_tags %}
 <table class="listing {% if full_width %}full-width{% endif %} {% block table_classname %}{% endblock %}">
-    {% if show_ordering_column %}
-        <col width="65px" />
-    {% endif %}
+    <col width="10px" />
```

Comment:
Now this no longer depends on a variable from Django, I think it would be nicer to move this into css.